tests/test_0001_html.py

```diff
@@ -258,3 +258,86 @@ def test_attribute_list():
     d.id_list.toggle('foo')
 
     assert d.id_list == ['foo', 'bar']
+
+
+@pytest.mark.dependency()
+def test_html_strings():
```

Comment:
nit: I think it's better to split this to several small tests (you already did it using `######`).
Also it's better to move those tests to new `test_0002_parsing_html.py`.

Reply:
@maratori I like that parsing is only one test, because if you break it, only one test fails instead of multiple. In big test-suites its often the case that you change one thing and the whole suite goes red. You can't  see hot-spots that easy. I tried to put this test not into its own file because pytest-dependency is not very clever when searching for tests in other modules.